### PR TITLE
Added a Three.js Course

### PR DESCRIPTION
**Pull request by CodeSudo**
Three.js course on youtube

## What does this PR do?
Add resource
### Description
Three.js course by Wael Yasmina on youtube
### Why is this valuable (or not)?
One of the in depth course for three.js
### How do we know it's really free?
Its Freely Available on Youtube
### For book lists, is it a book? For course lists, is it a course? etc.
Its a course
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -1054,6 +1054,7 @@
 
 #### Three.js
 
+* [Three.js Guide For Absolute Beginners](https://www.youtube.com/playlist?list=PLjcjAqAnHd1EIxV4FSZIiJZvsdrBc1Xho) - Wael Yasmina
```
Comment (sohansai) [inline courses/free-courses-en.md:1057]: Keep the playlist name consistent with the one provided in the link as you've used a different name

**Review by CodeSudo — COMMENTED** [review]
Updated the Title

File: courses/free-courses-en.md (hunk 1 above)
Comment (CodeSudo) [reply courses/free-courses-en.md]: Facing some lint issues let me fix this as well

**Review by CodeSudo — COMMENTED** [review]
Fixed the lint issue

**Pull request merged**